src/ProjectDependencies/App.xaml

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version = "1.0" encoding = "UTF-8" ?>
```

Comment:
i will not pretend to understand any of the maui bits, but dang thats a lot of boilerplate 😆